neurolib/models/multimodel/builder/base/params.py

```diff
@@ -0,0 +1,73 @@
+"""
```

Comment:
this file just implements functions for transforming float parameters into `sp.Symbol`